server/app/services/question/ActiveAndDraftQuestions.java

```diff
@@ -100,6 +100,27 @@ private ActiveAndDraftQuestions(VersionRepository repository) {
                     }));
   }
 
+  public ImmutableList<QuestionDefinition> getActiveAndDraftQuestions() {
```

Comment:
That does seem to be doing the same thing! I added a method to `ActiveAndDraftQuestions` since we already have that loaded in `ProgramIndexView` so I figured using the active and draft questions from there won't result in another db call. I also figured it would be handy to have a method that got all the questions from `ActiveAndDraftQuestions`. `ReadOnlyCurrentQuestionServiceImpl` isn't loaded in `ProgramIndexView`, so using that method would result in extra db calls we don't need I think. Do you think that's worth it to avoid duplicating logic? Regardless, I agree about the renaming.

Reply:
done!